src/electionguard/logs.py

```diff
@@ -16,12 +16,17 @@ class ElectionGuardLog(Singleton):
     """
 
     __logger: logging.Logger
+    __stream_handler: logging.StreamHandler
 
     def __init__(self) -> None:
         super().__init__()
 
         self.__logger = logging.getLogger("electionguard")
-        self.__logger.addHandler(get_stream_handler())
+        # Pythong's logger will use the most restrictive of the logger level and the handler level,
+        #   so set the logger to the lowest level the handler ever might log at
+        self.__logger.setLevel(logging.DEBUG)
+        self.__stream_handler = get_stream_handler(logging.INFO)
```

Comment:
Because of the bug above EG was asking for INFO but only getting ERROR.  My bug fix above may mean that EG is now too verbose.  May need to adjust this ^ if so.